service/history/historyEngine_test.go

```diff
@@ -5187,6 +5236,9 @@ func copyActivityInfo(sourceInfo *persistence.ActivityInfo) *persistence.Activit
 		MaximumInterval:          sourceInfo.MaximumInterval,
 		ExpirationTime:           sourceInfo.ExpirationTime,
 		MaximumAttempts:          sourceInfo.MaximumAttempts,
+		NonRetriableErrors:       sourceInfo.NonRetriableErrors,
+		//// Not written to database - This is used only for deduping heartbeat timer creation
+		// LastHeartbeatTimeoutVisibility: sourceInfo.LastHeartbeatTimeoutVisibility,
```

Comment:
remove?

Reply:
nop, 
1. this is test code
2. i want to put this here for reference, since previous code does not copy / paste everything

Reply:
why //// not //?